wirelength_analyzer/xcvup_device_data.py

```diff
@@ -97,6 +108,7 @@ def __contains__(self, item):
             (re.compile(r'INT_NODE_GLOBAL_\d{1,2}_INT_OUT[01]'),     0),
             (re.compile(r'IMUX_[EW]\d{1,2}'),                        0),
             (re.compile(r'IMUX_(CMT_)?XIPHY\d{1,2}'),                0),
+            (re.compile(r'IMUXOUT\d{1,2}'),                          0),
```

Comment:
Can you give an example of an IMUXOUT pip?

Reply:
see the switchbox in the tile `INT_INTF_L_PCIE4_X100Y7`

Reply:
One example is `INT_INTF_L_PCIE4_X100Y7/INT_INTF_L_PCIE4.DELAYWIRE_PCIE->IMUXOUT0`.